client/src/components/button/RoundButton.jsx

```
@@ -1,6 +1,11 @@
 import React from 'react';
+import clsx from 'clsx';
 import styles from './RoundButton.module.css';
 
-export default function RoundButton({content}) {
-  return <button className={styles.btn}>{content}</button>;
+export default function RoundButton({ children, color = 'blue', clickHandler }) {
+  return (
+    <button onClick={clickHandler} className={clsx(styles.btn, styles[color])}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)